test/verify/storagelib.py

```diff
@@ -217,6 +217,28 @@ def dialog_set_val(self, field, val):
             else:
                 self.browser.set_val(self.dialog_field(field), val)
 
+    # Workaround: React does not fire onChange event when setting input.value attribute directly
+    # https://github.com/facebook/react/issues/8971
+    # potential fix: https://codepen.io/pudgereyem/live/OWBrdv
+    def dialog_type_val(self, field, val):
```

Comment:
The `dialog_set_val` function needs to be able to handle all types of fields, as it is used also by the high-level `dialog` and `dialog_with_retry` functions.  And unfortunately, it needs to handle both React and the old Mustache dialogs at the same time... Right now only check-storage-luks runs into React dialogs, but the goal is to change all dialogs to be implemented with React, and thus `dialog` and `dialog_with_retry` should be ready to work with them.

Does using React really change the behavior of the `<input>` element so much?  Could we maybe synthesize the `onChange` event?